drakrun/drakrun/draksetup.py

```diff
@@ -245,6 +246,86 @@ def send_usage_report(report):
         logging.exception("Failed to send usage report. This is not a serious problem.")
 
 
+def create_rekall_profiles(install_info):
```

Comment:
Can we make it in such way that usermode profile generation could be optionally disabled using some `--usermode=no` switch or sth?

Reply:
Fixed in https://github.com/CERT-Polska/drakvuf-sandbox/pull/148/commits/d9d12019583ec022ca3673d7fb9221588e82b28e